### PR TITLE
test(snapshots): remove classes from snapshots

### PR DESCRIPTION
**Pull request by dcwither**
### Summary:
Pulled `https://github.com/chanzuckerberg/along/blob/main/client/src/test/removeStylesJestSerializer.ts` from along - we should consider moving this into https://github.com/chanzuckerberg/frontend-libs

### Test Plan:
`npm test`

### CODE REVIEW

**Comment by dcwither** [comment]
https://github.com/chanzuckerberg/lp-design-system/pull/247/checks?check_run_id=1573354928#step:4:29 Noooo!

I think this has to do with changing dependencies and busting the dependencies cache

File: .eslintrc [hunk 1]
```diff
@@ -35,7 +35,7 @@
       "extends": ["plugin:@typescript-eslint/recommended"]
     },
     {
-      "files": ["*.spec.ts", "*.spec.tsx"],
+      "files": ["*.spec.ts", "*.spec.tsx", "**/test/**"],
```
Comment (dcwither) [inline .eslintrc:38]: test folder is supposed to be only for test utils

**Review by ahuth — APPROVED** [review]
Nice, LGTM

**Comment by ahuth** [comment]
> we should consider moving this into https://github.com/chanzuckerberg/frontend-libs

Makes sense. I would like to look into getting rid of the jest-styled-components dependency when we do that.

**Comment by anniehu4** [comment]
woot! task for tracking the followup: https://app.clubhouse.io/connections/story/62274/move-removestylesjestserializer-to-https-github-com-chanzuckerberg-frontend-libs

**Pull request merged**